seam/seam.py

```diff
@@ -24,6 +26,8 @@ def __init__(
         workspace_id: Optional[str] = None,
         endpoint: Optional[str] = None,
         wait_for_action_attempt: Optional[Union[bool, Dict[str, float]]] = False,
+        client: Optional[requests.Session] = None,
+        client_options: Optional[Dict[str, Any]] = None,
```

Comment:
Same note on removing for now.

Reply:
https://github.com/seamapi/python-next/pull/63/commits/8ba79cfccfbb02571f183e01ba00d45183984d2c